app/views/kudos/index.html.erb

```diff
@@ -13,8 +13,11 @@
     </div>
     <div class="card-footer">
       <%= link_to 'Show', kudo, class: "btn btn-outline-dark btn-sm" %>
-      <%= link_to 'Edit', edit_kudo_path(kudo), class: "btn btn-outline-secondary btn-sm" %>
-      <%= link_to 'Destroy', kudo, method: :delete, data: { confirm: 'Are you sure?' }, class: "btn btn-outline-danger btn-sm" %>
+      <% if current_employee == kudo.giver %>
+        <%= link_to 'Edit', edit_kudo_path(kudo), class: "btn btn-outline-secondary btn-sm" %>
```

Comment:
A co się stanie jeżeli nie-giver, wejdzie w adres edycji kudo albo usunięcia kudo wpisując adres w przeglądarkę? Musisz dodać zabezpieczenie w kontrolerze które nie pozwoli usuwać i edytować kudosów przez nie-giverów

Reply:
Faktycznie, nie było to bezpieczne rozwiązanie. Aktualnie dodałem zabezpieczenie w kontrolerze przed edycją i usunięciem. Dziękuję za podpowiedź